crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -425,18 +425,20 @@ impl Device {
         session: InboundGroupSession,
         message_index: Option<u32>,
     ) -> OlmResult<(Session, Raw<ToDeviceEncryptedEventContent>)> {
-        let (event_type, content) = {
+        let content = {
             let export = if let Some(index) = message_index {
                 session.export_at_index(index).await
             } else {
                 session.export().await
             };
             let content: ForwardedRoomKeyContent = export.try_into()?;
 
-            (content.event_type(), content)
+            content
         };
```

Comment:
```suggestion

        let content: ForwardedRoomKeyContent = {
            let export = if let Some(index) = message_index {
                session.export_at_index(index).await
            } else {
                session.export().await
            };
            export.try_into()?
        };

Reply:
Wouldn't let me commit the suggestion, but applied manually in 042925eb435e49ed10ff7a090e84f73e00dfb271.